### ISSUE TITLE
Polaris USB not working

### ISSUE THREAD

**Issue by mikotalik** [edk2-porting/WOA-Drivers#88]
Hello. After 2 full reinstalls, USB on my Xiaomi Mi Mix 2s still doesn't work. I cannot connect mouse, keyboard, or usb flash disk. If I boot with some of the devices attached, I cannot even start Windows. Charging works perfectly.

**Comment by cocus** [comment]
What's the old driver pack that "works"? I have installed it recently with the latest edk2-msm (https://github.com/edk2-porting/edk2-msm/releases/tag/2301.1) and latest WOA-Drivers (2210.1-fix), and usb doesn't work (it can charger, but nothing gets detected when I plug a usbc OTG). Funny enough, I see on the device manager that a new device shows up (a usb root hub controller) when I plug the adapter, but nothing gets enumerated. I don't know if something is not turning on the power to the devices or if there's something else going on. I'd like to NOT re-install, is it possible to just use "the old driver release"? If so, can I just downgrade the usb or the drivers related to this issue?

**Comment by elsa-d55503** [comment]
> What's the old driver pack that "works"? I have installed it recently with the latest edk2-msm (https://github.com/edk2-porting/edk2-msm/releases/tag/2301.1) and latest WOA-Drivers (2210.1-fix), and usb doesn't work (it can charger, but nothing gets detected when I plug a usbc OTG). Funny enough, I see on the device manager that a new device shows up (a usb root hub controller) when I plug the adapter, but nothing gets enumerated. I don't know if something is not turning on the power to the devices or if there's something else going on. I'd like to NOT re-install, is it possible to just use "the old driver release"? If so, can I just downgrade the usb or the drivers related to this issue?

It may be too late now, but USB-OTG worked when I used v2.0rc2 when I installed it.

**Comment by cocus** [comment]
> It may be too late now, but USB-OTG worked when I used v2.0rc2 when I installed it.

Yes, I've used the old one. Not sure what changed. But certainly nothing got updated for polaris even after 2210.1-fix :(

**Comment by jorbig** [comment]
Maybe this is of help: https://github.com/n00b69/woa-polaris/blob/main/guide/materials.md#toggling-usb-host-mode
